### PR TITLE
[SDK-2798] - React Native push notifications update

### PR DESCRIPTION
**Pull request by wesleyorbin**

### CODE REVIEW

**Comment by davidbielik** [comment]
should we make `useBrazeIosPush` and `enableFirebaseCloudMessaging` consistent in naming? like `enableBrazeIosPush` or `useFirebaseCloudMessaging`?

File: _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md [hunk 1]
```diff
@@ -14,10 +14,72 @@ Integrating push notifications in React Native requires setting up each native p
...
+      ],
+    ]
+  }
+}
+```
+
+{% endtab %}
+{% tab Android %}
+
+Follow the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration/).
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md:46]: I would just remove this. These steps are basically all native and wouldn't be easily done via Expo

**Review by radixdev — APPROVED** [review]

**Comment by wesleyorbin** [comment]
> should we make `useBrazeIosPush` and `enableFirebaseCloudMessaging` consistent in naming? like `enableBrazeIosPush` or `useFirebaseCloudMessaging`?

done

**Review by davidbielik — DISMISSED** [review]

File: _docs/_developer_guide/platform_integration_guides/react_native/content_cards.md [hunk 2]
```diff
@@ -15,6 +15,23 @@ The Braze SDKs include a default card feed to get you started with Content Cards
 
 ## Customization
 
+To build your own UI, you can get a list of available cards, and listen for updates to cards:
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/react_native/content_cards.md:18]: @wesleyorbin i added this. lmk if this doesn't look right.

Comment (wesleyorbin) [reply _docs/_developer_guide/platform_integration_guides/react_native/content_cards.md:18]: LGTM

File: _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md [hunk 3]
```diff
@@ -14,10 +14,72 @@ Integrating push notifications in React Native requires setting up each native p
...
+
+{% endtab %}
+{% tab iOS %}
+
+Follow the [iOS integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/ios/push_notifications/integration/). If you prefer not to request push permission upon launching the app, you should omit the `requestAuthorizationWithOptions:completionHandler:` call in your AppDelegate and follow the step below.
+
+{% endtab %}
+{% endtabs %}
+
+## Step 2: Request Push Notifications Permission
```
Comment (KellieHawks) [inline _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md:56]:
```suggestion
## Step 2: Request push notifications permission
```

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**